### PR TITLE
Newsletters: add 240 (2023-03-01)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic links @harding 
- [x] Bitcoin Core 25943 @glozow 
- [x] Bitcoin Core 26595 @bitschmidty

### CODE REVIEW

**Review by LarryRuane — COMMENTED** [review]
LGTM, I verified the Bitcoin Core PR numbers

File: _posts/en/newsletters/2023-03-01-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25943][] adds a parameter to the sendrawtransaction
```
Comment (azuchi) [inline _posts/en/newsletters/2023-03-01-newsletter.md:79]:
```suggestion
- [Bitcoin Core #25943][] adds a parameter to the `sendrawtransaction`
```

File: _posts/en/newsletters/2023-03-01-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+---
+title: 'Bitcoin Optech Newsletter #240'
+permalink: /en/newsletters/2023/03/01/
+name: 2023-03-22-newsletter
```
Comment (azuchi) [inline _posts/en/newsletters/2023-03-01-newsletter.md:4]:
```suggestion
name: 2023-03-01-newsletter
```

File: _posts/en/newsletters/2023-03-01-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  to the discussion about a draft BIP for Codex32 (see [last week's
+  newsletter][news239 codex32]), a scheme that allows creating,
+  verifying, and using recovery codes for a [BIP32][topic bip32] seed.
+  A particular advantage of Codex32 over existing schemes is the ability
+  to verify the integrity of backups using just pen, paper,
+  documentation, and a modest amount of time.
+
+    As designed, Codex32 provides very strong assurances about its
+    ability to detect errors in the backups.  Peter Todd suggested a far
+    easier method would be to generate recovery codes that could have
```
Comment (jirijakes) [inline _posts/en/newsletters/2023-03-01-newsletter.md:28]: There seem to be some words missing here.

**Comment by harding** [comment]
Made edits for all feedback (thanks @LarryRuane and @azuchi !), reviewed contributions from @glozow and @bitschmidty (thanks!), and added lede, releases/RCs, and topic links.

@bitschmidty it's up to you, but I'd be happy to see an announcement for the new podcast section of the site.  The way we've [previously](https://bitcoinops.org/en/newsletters/2019/08/21/#new-optech-compatibility-matrix) announced [new site features](https://bitcoinops.org/en/newsletters/2019/11/13/#new-topic-index-on-optech-website) is with an item at the bottom of the News section, so I'd suggest that if you want to do it.  I'd also be happy to include it in next week's newsletter if you want me to write the announcement.  Note that you'll probably need to rebase this PR on master to get the link checker test to pass if you link to the podcasts page (or you can just merge it live).

Thanks everyone!

**Comment by bitschmidty** [comment]
@harding good point. With the podcast going live near the newsletter day, I suggest we announce in the newsletter next week, including a blog post (like [topics](https://bitcoinops.org/en/topics-announcement/) announcement).

I can repurpose the twitter thread content for the blog post or if you have something in mind, happy to review that as well.

**Review by bitschmidty — COMMENTED** [review]
Pushed a small formatting change to Bitcoin Core #26595 summary

ACK the rest

**Comment by bitschmidty** [comment]
Squashed and merged! Thank you @harding and @glozow for authoring and @azuchi and @LarryRuane for the reviews!

**Pull request merged**